pulpcore/pulpcore/app/models/publication.py

```diff
@@ -107,21 +107,36 @@ def delete(self, **kwargs):
             CreatedResource.objects.filter(object_id=self.pk).delete()
             super().delete(**kwargs)
 
+    def update_distributions(self, model=None):
+        """
+        Update distributions with this publication.
+
+        This supports the auto-distribution feature by update the publication on
```

Comment:
How about changing this to: "Plugins that provide their own distribution models can call this method to implement the auto-distribution feature. This method has to be called after a publication has been saved to the database and marked as 'complete'. "